webapp/webapp.v3.spec.yaml

```diff
@@ -670,6 +674,10 @@ components:
           example: '2018-04-05T01:23:45+02:00'
           format: date-time
           description: Return only repositories changed after the given date
+        third_party:
+          type: boolean
+          enum: [ true, false ]
+          description: Return third party content (Content not managed by RedHat)
```

Comment:
I would add the info that param is optional and false by default.

Reply:
Added